.env.example

```
@@ -0,0 +1,3 @@
+GITHUB_TOKEN
+GITHUB_REPOSITORY_OWNER
+GITHUB_REPOSITORY_NAME
```

Makes the format slightly clearer:

```suggestion
GITHUB_TOKEN=
GITHUB_REPOSITORY_OWNER=
GITHUB_REPOSITORY_NAME=
```